packages/strapi-design-system/src/Text/index.js

```diff
@@ -34,13 +35,15 @@ const textStyles = {
   },
 };
 
-export const Text = styled.p`
+export const Text = styled.span`
   font-weight: ${({ highlighted }) => (highlighted ? 500 : 400)};
   font-size: ${({ small }) => textStyles[small ? 'S' : 'M'].fontSize};
   line-height: ${({ small }) => textStyles[small ? 'S' : 'M'].lineHeight};
   color: ${handleColor};
 `;
 
+export const Paragraph = (props) => <Text as="p" {...props} />;
```

Comment:
I'm afraid it's going to get long to write `Paragraph` every time. WDYT of renaming that one to `<P />`?

Reply:
Yes sure